architecture/core/release-service.md

```diff
@@ -5,6 +5,12 @@ eleventyNavigation:
   parent: Core Services
   order: 4
 toc: true
+local_summary:
+  scope: "Release orchestration, privileged pipelines, cross-namespace releases"
+  key_crds: "Release, ReleasePlan, ReleasePlanAdmission, ReleaseStrategy"
```

Comment:
ReleaseStrategy no longer exists. Does this still have to be here because historically it existed, or can it be removed?

Reply:
This page should be cleaned up/updated to remove the references: https://github.com/konflux-ci/architecture/blob/main/architecture/core/release-service.md?plain=1#L23-L24